doc/modules/preprocessing.rst

```diff
@@ -426,6 +426,47 @@ features, one has to explicitly set ``n_values``. For example,
 See :ref:`dict_feature_extraction` for categorical features that are represented
 as a dict, not as integers.
 
+.. _discretization:
+
+Discretization
+==============
+
+`Discretization <https://en.wikipedia.org/wiki/Discretization_of_continuous_features>`_
+provides a way to quantize continuous features into discrete values. Certain
+datasets with many continuous features may benefit from discretization, because
+discretization can transform the dataset of continuous attributes to one with
+only nominal attributes.
+
+Scikit-learn provides a mechanism way for users to discretize a feature into
```

Comment:
Topicalise KBinsDiscretizer: "KBinsDiscretizer discretizes features into k bins of equal width".